src/domain/user.js

```diff
@@ -174,8 +174,8 @@ export default class User {
     return User._findByUnique('id', id)
   }
 
-  static async findManyByFirstName(firstName) {
-    return User._findMany('firstName', firstName)
+  static async findManyByFirstName(key, value) {
```

Comment:
The purpose of this function is to provide a simple interface to find users by their first name. This change is making the function more general, which is what `_findMany` already does. I don't think this is a useful change to make.

If you want to add functionality to be able to search users by multiple fields, you should create new things for that and use the existing functionality where you're able to

Reply:
Changed the functionality of the functions findManyByFirstName and getAllByFirstName